src/components/text-ellipsis/TextEllipsis.tsx

```diff
@@ -56,9 +53,10 @@ export const TextEllipsis: React.FC<TextEllipsisProps> = ({
     return(
       <Tooltip
         description={ children as JSX.Element }
-        placement={tooltipPlacement || 'bottom'}
+        placement={ 'top' }
```

Comment:
The previous solution looked better because you can clearly see that you can pass in a value for it, or it will default to 'bottom'. This one looks like it's always set to 'top'. Even though you can overwrite it, but just by looking at the code of this component. It's quite unclear.

Reply:
Not sure where my comment disappeared or if I never saved it. If okay, I'll leave it like this, didn't wanna break out individual props from Tooltip.